styleguide/screens/Patterns/Cards/Cards.js

```diff
@@ -0,0 +1,656 @@
+          </T>
+
+          <H level={ 3 } className={ m.mtLgIi }>Default</H>
+          <T elm="p" className={ m.mtr }>Normally, the card component does absolutely nothing.</T>
+          <Specimen
+            classNames={ {
+              root: m.mtr,
+              specimenContainer: m.par,
+            } }
+            code={ dedent`
+                <Card>
+                  // Card content
+                </Card>
+            ` }
+          >
+            <div className={ css.cardContainer }>
+              <div className={ css.cardContainer }>
+                <Card className={ css.card }>
+                  <div className={ css.defaultCardContent }>
+                    Default
+                  </div>
+                </Card>
+              </div>
+            </div>
+          </Specimen>
+          <H level={ 3 } className={ m.mtLgIi }>As a link</H>
+          <T elm="p" className={ m.mtr }>
+            When provided a href prop, the card adds a hover effect,
+            which moves the card further off the page than it&#39;s surrounding elements.
+          </T>
+          <Specimen
+            classNames={ {
+              root: m.mtr,
+              specimenContainer: cx(m.par, css.specimenContainer),
+            } }
+            code={ dedent`
+              <Card href="#">
+                // Card content
+              </Card>
+            ` }
+          >
+
+            <div className={ css.cardContainer }>
+              <Card href="#" className={ css.card }>
+                <div className={ css.defaultCardContent }>
+                  With a link
+                </div>
+              </Card>
+            </div>
+          </Specimen>
+        </D>
+        <D>
+          <H level={ 2 } className={ shared.componentTitle }>Picture Card</H>
+          <T elm="p" className={ m.mtr }>
+            Similarly to the <C>Card</C> component, the picture card is
+            the foundation of other card components. It&#39;s typically
+            used where the background of the card should be a full
+            bleed image. Otherwise, it acts exactly as the regular card component.
+          </T>
+          <Specimen
+            classNames={ {
+              root: m.mtr,
+              specimenContainer: m.par,
+            } }
+            code={ dedent`
+              <PictureCard
+                src="https://source.unsplash.com/random"
+              >
+                // PictureCard content
+              </PictureCard>
+            ` }
+          >
+            <div className={ css.cardContainer }>
+              <PictureCard
+                src="https://source.unsplash.com/random"
+                className={ css.pictureCard }
+              >
+                PictureCard
+              </PictureCard>
+            </div>
+          </Specimen>
+        </D>
+        <D>
+          <H level={ 2 } className={ shared.componentTitle }>Space Feature Card with Link</H>
```

Comment:
Do we have a version without the link? I feel like feature cards should _always_ have one